README.md

```diff
@@ -291,7 +324,7 @@ makes it an attractive option for situations where dependency management
 is expensive (e.g., an R application running in a browser via
 [WebAssembly](https://docs.r-wasm.org/webr/latest/)).
 
-[^1]: At this point, experienced base plot users might protest that you
+1.  At this point, experienced base plot users might protest that you
```

Comment:
Manual override here and the second footnote below. Don't worry if that's a pain, though. I can always knit on my side over this is merged. Just lmk.